nncf/torch/model_creation.py

```diff
@@ -156,6 +160,34 @@ def create_compressed_model(
     return compression_ctrl, compressed_model
 
 
+def get_input_info_from_config(config: NNCFConfig) -> ModelInputInfo:
+    if has_input_info_field(config):
+        return FillerInputInfo.from_nncf_config(config)
+
+    nncf_logger.debug(
+        "Config has no 'input_info' section, trying to use dataloader output as model inputs " "for graph building."
+    )
+    exact_info = ExactInputsInfo.from_nncf_config_dataloaders(config)
+    if exact_info is not None:
+        return exact_info
+    raise RuntimeError(
+        "Could not determine tensor inputs for the model's forward call.\n"
+        "If you are using the `nncf.quantize` API, make sure that you supply the "
+        "calibration dataloader to the `nncf.quantize` call.\n"
+        "If you are using the `create_compressed_model` API, either specify the "
+        "inputs by using the 'input_info' section in the NNCFConfig, or attach an "
+        "initialization dataloader to the NNCFConfig by calling "
+        "`NNCFConfig.register_extra_structs(...)` with one of the following extra "
+        f"structures:\n"
+        f"{EXTRA_STRUCTS_WITH_DATALOADERS}\n"
+        f"or by calling `nncf.torch.register_default_init_args`"
+    )
+
+
+def create_nncf_network_with_inputs_from_config(model: torch.nn.Module, config: NNCFConfig):
+    return create_nncf_network(model, config=config)
```

Comment:
Could you please provide more information about why this method is required?

Reply:
Another leftover, thanks for spotting.